src/SuperDumpService/Program.cs

```diff
@@ -5,7 +5,7 @@ namespace SuperDumpService {
 	public static class Program {
 		public static void Main(string[] args) {
 			var host = new WebHostBuilder()
-				.UseKestrel()
+				.UseKestrel(opt => opt.Limits.MaxRequestBodySize = 8589934592L)  // 8gb
```

Comment:
Well, in that case using long.MaxValue sounds like the better choice. Not sure though, if it introduces any kind of security problems?